### PR TITLE
fixed password hash

### PR DESCRIPTION
**Pull request by aeydie**
Submitting the new code that hashes the passwords in our database for review!

### CODE REVIEW

**Review by lfallert — APPROVED** [review]
created 2 accounts, each had 2 distinct password hashes saved in database, well done! When we merge we will need to make sure we keep the imports

File: src/main/java/com/launchcode/AroundTownServer/controllers/UserController.java [hunk 1]
```diff
@@ -3,11 +3,8 @@
 import com.launchcode.AroundTownServer.data.UserRepository;
 import com.launchcode.AroundTownServer.models.User;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
```
Comment (mertzjl91) [inline src/main/java/com/launchcode/AroundTownServer/controllers/UserController.java]: Could we delete this blank line, to keep the imports together in one chunk?

Comment (lfallert) [reply src/main/java/com/launchcode/AroundTownServer/controllers/UserController.java]: I agree, done and pushed

**Review by aeydie — COMMENTED** [review]
It looks like my code is correct now from what I'm seeing here, let me know if I'm missing something?

**Review by CNichole726 — APPROVED** [review]
I was able to create 3 new users with encoded passwords saved to the database, nice job!

**Pull request merged**